Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN apk add --no-cache \
     openssh-client \
     rsync \
     build-base \
-    libc6-compat
+    libc6-compat \
+    npm && \
+    npm install postcss-cli
```

Comment:
Autoprefixer is missing as well here. See my suggestion ([copy from contributor-site](https://github.com/kubernetes-sigs/contributor-site/blob/028e025e3bcd3af87004bf6e7390c69d1f4a76b1/Dockerfile#L15-L18) which also includes a comment on why the components are installed).

```suggestion
    npm
    
# Required for PostCSS
RUN npm install -G \
    autoprefixer \
    postcss-cli
```